pkg/component/etcd/bootstrap.go

```diff
@@ -250,6 +255,29 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 			},
 		}
 
+		service = &corev1.Service{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      druidServiceName,
+				Namespace: b.namespace,
+				Labels: utils.MergeStringMaps(map[string]string{
+					resourcesv1alpha1.HighAvailabilityConfigType: resourcesv1alpha1.HighAvailabilityConfigTypeController,
+				}, labels()),
+			},
+			Spec: corev1.ServiceSpec{
+				Type:            corev1.ServiceTypeClusterIP,
+				SessionAffinity: corev1.ServiceAffinityNone,
```

Comment:
`SessionAffinityNone` is the default. Do we really need to specify it here? We also do not specify other default values, e.g. `internalTrafficPolicy`.

Reply:
Removed `SessionAffinityNone`